core/templates/App.ts

```diff
@@ -25,6 +25,7 @@
  */
 import 'firebase/auth';
 import 'hammerjs';
+import 'leaflet/dist/leaflet.css';
```

Comment:
This is only required for the exploration-editor//play page, right? so should we import it in those modules?